custom_components/tuya_local/devices/airquality6in1.yaml

```diff
@@ -0,0 +1,70 @@
+name: Air quality monitor 6 in 1
+products:
+  - id: ugjbnqj2mffaexp5
+    name: Air Housekeeper
+primary_entity:
+  entity: sensor
+  name: PM2.5
+  class: volatile_organic_compounds
+  icon: "mdi:factory"
+  dps:
+    - id: 2
+      type: integer
+      name: sensor
+      unit: µg/m³
+      class: measurement
+      mapping:
```

Comment:
Mapping is not needed here, as a scale of 1 is default. 

However, if that scale is copied directly from the iot.tuya.com docs, then the actual scale is 10, as the Tuya definition of scale seems to be how many decimal places to shift.